lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,67 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
+  option "--clean", :flag, "clean up the generated dump of plugins", :default => false
+  option "--override", :flag, "Override a previously generated package file", :default => false
+
+  def execute
+    signal_error("File #{LogStash::Environment::GEMFILE_PATH} does not exist or is not writable, aborting") unless File.writable?(LogStash::Environment::GEMFILE_PATH)
+    puts("packaging plugins for offline usage")
```

Comment:
Can we add a message to tell the user the location of the generated package?
`Generated at: #{location}` ?